homework_25/src/main/java/org/jdbc/operations/LessonDaoOperations.java

```diff
@@ -0,0 +1,15 @@
+package org.jdbc.operations;
+
+import org.jdbc.entity.Lesson;
+
+import java.util.List;
+
+public interface LessonDaoOperations<T, ID> {
```

Comment:
If you wanna use generics, then rename this interface to `GenericDaoOperations` and exclude `lesson` from all method names. Otherwise, get rid of generics and set Lesson instead of T and Long instead of ID